Cargo.toml

```diff
@@ -29,7 +29,7 @@ hlua = "0.4.1"
 aabb-quadtree = "0.1.0"
 zstd = "0.4.14"
 stopwatch = "0.0.7"
-atomic = {version = "0.4", features = ["nightly"]}
+atomic = {version = "= 0.4.0", features = ["nightly"]}
```

Comment:
This should read:
```
atomic = {version = "0.4.0", features = ["nightly"]}
```
instead, right?

Reply:
Unfortunately not. According to [this document](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html), `"0.4.0"` would include version `0.4.1`, which is the version I am trying to exclude as it seems to me to be broken on latest nightly. (I originally specified `"0.4.0"` and had issues because it was using `0.4.1`.) Try `0.4.1` if you like (make sure it's updated in your lock file) and see if it works for you on latest nightly. It's possible that something is wrong in my end.